xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.cpp

```diff
@@ -58,6 +58,8 @@ void VideoPicture::Reset()
 
   iWidth = 0;
   iHeight = 0;
+  iXOffset = 0;
```

Comment:
Please try to avoid Hungarian notation as per our code guidelines (xOffset and yOffset would suffice)

Reply:
fixed in https://github.com/xbmc/xbmc/pull/24431/commits/5ca5989c1cb83f91c40f7c2e9ff4a3eceeae4e0f